internal/scte35.go

```diff
@@ -0,0 +1,83 @@
+package internal
+
+import (
+	"github.com/Comcast/gots/v2/scte35"
+)
+
+type SCTE35Info struct {
+	PID           uint16                   `json:"pid"`
+	SpliceCommand SpliceCommand            `json:"spliceCommand"`
+	SegDesc       []SegmentationDescriptor `json:"segmentationDes,omitempty"`
+}
+
+type SpliceCommand struct {
+	Type     string `json:"type"`
+	EventId  uint32 `json:"eventId,omitempty"`
+	PTS      uint64 `json:"pts,omitempty"`
```

Comment:
Don't use omit empty without a pointer. Otherwise a zero-value will not be output.
In general, this value should be available for a CUE-OUT.